README.md

```diff
@@ -4,9 +4,15 @@
 <img src="https://github.com/kubernetes/kubernetes/raw/master/logo/logo.png"  width="100x"><a href="https://aws.amazon.com/opensource/"><img width="192x" src="https://d0.awsstatic.com/logos/powered-by-aws.png" alt="Powered by AWS Cloud Computing"></a>
 </p>
 <p align="center">
+<!-- build passing badge -->
+<a href="https://prow.k8s.io/?job=post-cluster-api-provider-aws-push-images">
+<img alt="Build Status" src="https://prow.k8s.io/badge.svg?jobs=post-cluster-api-provider-aws-push-images"></a>
```

Comment:
`https://prow.k8s.io/?job=post-cluster-api-provider-aws-push-image` is not showing any jobs.
Also it is not the build job.

Reply:
Today we are not seeing any build list for any provider on prow jobs. I checked in CAPI repository as well. CAPI's build list is also empty `https://prow.k8s.io/?job=post-cluster-api-push-images`.

I will post my findings once I will get some more hints on why it's happening

Reply:
I have removed `build passing` badge  as the core of our tests are in prow and we might get same `build | no results` multiple times, so feels it's not good to show it on repo